macaroonbakery/utils.py

```diff
@@ -111,17 +112,21 @@ def cookie(
         expires=None):
     '''Return a new Cookie using a slightly more
     friendly API than that provided by six.moves.http_cookiejar
+
     @param name The cookie name {str}
     @param value The cookie value {str}
     @param url The URL path of the cookie {str}
-    @param expires The expiry time of the cookie {datetime}
+    @param expires The expiry time of the cookie {datetime}. If provided,
+        it must be a naive timestamp.
```

Comment:
it must be a naive timestamp in UTC.

?

Reply:
done